.github/workflows/update-coverage.yml

```diff
@@ -0,0 +1,36 @@
+name: Update Coverage in README
+on:
+  push:
+    branches:
+      - main
+jobs:
+  update-coverage-in-readme:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
There is a actions/checkout@v4 is it possible to use this one also?

Reply:
Yes, thank you, I changed that!